tests/topotests/static_routing_with_ibgp/test_static_routes_topo2_ibgp.py

```diff
@@ -0,0 +1,2357 @@
+            stop_router(tgen, "r2")
+            sleep(5)
+            start_router(tgen, "r2")
+
+            step("After reloading, verify that routes are still present in R2.")
+            result = verify_rib(
+                tgen,
+                addr_type,
+                dut,
+                second_rte,
+                next_hop=nh,
+                protocol=protocol,
+                fib=True,
+            )
+            assert result is True, (
+                "Testcase {} : Failed \nError: Route "
+                " is missing in RIB".format(tc_name)
+            )
+
+        dut = "r2"
+        for intfr in range(0, 8):
+            intf = topo["routers"]["r2"]["links"]["r3-link{}".format(intfr)][
+                "interface"
+            ]
+            shutdown_bringup_interface(tgen, dut, intf, False)
+
+        dut = "r3"
+        for intfr in range(0, 8):
+            intf = topo["routers"]["r3"]["links"]["r2-link{}".format(intfr)][
+                "interface"
+            ]
+            shutdown_bringup_interface(tgen, dut, intf, False)
+        sleep(180)
+        step("verify bgp convergence as all interface are shut")
+
+        bgp_convergence = verify_bgp_convergence(tgen, topo)
+        assert bgp_convergence is not True, "Testcase {} : Failed \n"
+        "Error: {}".format(tc_name, bgp_convergence)
+
+        dut = "r2"
+        protocol = "static"
+        for addr_type in ADDR_TYPES:
+            for nhp in range(1, 9):
+                input_dict_4 = {
+                    "r2": {
+                        "static_routes": [
+                            {
+                                "network": PREFIX1[addr_type],
+                                "next_hop": NEXT_HOP_IP["nh" + str(nhp)][addr_type],
+                            }
+                        ]
+                    }
+                }
+            nh = NEXT_HOP_IP["nh1"][addr_type]
+            result = verify_rib(
+                tgen,
+                addr_type,
+                dut,
+                input_dict_4,
+                next_hop=nh,
+                protocol=protocol,
+                fib=True,
+            )
+            assert result is True, "Testcase {} : Failed \nError: Routes are"
+            " missing present in RIB".format(tc_name)
+
+        dut = "r3"
+        protocol = "bgp"
+        for addr_type in ADDR_TYPES:
+            input_dict_4 = {"r2": {"static_routes": [{"network": PREFIX1[addr_type],}]}}
+            nh = NEXT_HOP_IP["nh1"][addr_type]
+            result = verify_rib(
+                tgen,
+                addr_type,
+                dut,
+                input_dict_4,
+                next_hop=nh,
+                protocol=protocol,
+                expected=False,
+            )
+            assert result is not True, "Testcase {} : Failed \nError: Routes "
+            " are still present in RIB after BGP nbr is down".format(tc_name)
+
+        sleep(5)
```

Comment:
here too

Reply:
Changes made are:
1. Removed the static sleep.
2. Base config is modified to use bgp aggressive timers.